src/app_model/types/_keys/_key_codes.py

```diff
@@ -571,6 +585,48 @@ class _KM(NamedTuple):
         'cmd': KeyCode.Meta,
     }
 
+    # key symbols mappings per platform
+    WIN_KEY_SYMBOLS: dict[str, str] = {
+        "Ctrl": "Ctrl",
+        "Shift": "⇧",
+        "Alt": "Alt",
+        "Meta": "⊞",
+        "Left": "←",
+        "Right": "→",
+        "Up": "↑",
+        "Down": "↓",
+        "Backspace": "⌫",
+        "Delete": "⌦",
+        "Tab": "↹",
+        "Escape": "Esc",
+        "Return": "⏎",
+        "Enter": "↵",
+        "Space": "␣",
+    }
+    MACOS_KEY_SYMBOLS: dict[str, str] = {**WIN_KEY_SYMBOLS, "Ctrl": "⌃", "Alt": "⌥", "Meta": "⌘"}
+    LINUX_KEY_SYMBOLS: dict[str, str] = {**WIN_KEY_SYMBOLS, "Meta": "Super"}
+
+    # key names mappings per platform
+    WIN_KEY_NAMES: dict[str, str] = {
+        "Ctrl": "Ctrl",
+        "Shift": "Shift",
+        "Alt": "Alt",
+        "Meta": "Win",
+        "Left": "Left",
+        "Right": "Right",
+        "Up": "Up",
+        "Down": "Down",
+        "Backspace": "Backspace",
+        "Delete": "Supr",
+        "Tab": "Tab",
+        "Escape": "Esc",
+        "Return": "Return",
+        "Enter": "Enter",
+        "Space": "Space",
+    }
+    MACOS_KEY_NAMES: dict[str, str] = {**WIN_KEY_NAMES, "Ctrl": "Control", "Alt": "Option", "Meta": "Cmd"}
+    LINUX_KEY_NAMES: dict[str, str] = {**WIN_KEY_NAMES, "Meta": "Super"}
```

Comment:
definitely nit-picky, but we can save ourselves the `if os ==` conditionals on lookup by adding on more level to these maps... (then see associated change below)

```suggestion
    # key symbols on all platforms
    KEY_SYMBOLS: dict[KeyCode, str] = {
        KeyCode.Shift: "⇧",
        KeyCode.LeftArrow: "←",
        KeyCode.RightArrow: "→",
        KeyCode.UpArrow: "↑",
        KeyCode.DownArrow: "↓",
        KeyCode.Backspace: "⌫",
        KeyCode.Delete: "⌦",
        KeyCode.Tab: "⇥",
        KeyCode.Escape: "⎋",
        KeyCode.Enter: "↵",
        KeyCode.Space: "␣",
        KeyCode.CapsLock: "⇪",
    }
    # key symbols mappings per platform
    OS_KEY_SYMBOLS: dict[OperatingSystem, dict[KeyCode, str]] = {
        OperatingSystem.WINDOWS: {**KEY_SYMBOLS, KeyCode.Meta: "⊞"},
        OperatingSystem.LINUX: {**KEY_SYMBOLS, KeyCode.Meta: "Super"},
        OperatingSystem.MACOS: {
            **KEY_SYMBOLS,
            KeyCode.Ctrl: "⌃",
            KeyCode.Alt: "⌥",
            KeyCode.Meta: "⌘",
        },
    }

    # key names mappings per platform
    OS_KEY_NAMES: dict[OperatingSystem, dict[KeyCode, str]] = {
        OperatingSystem.WINDOWS: {KeyCode.Meta: "Win"},
        OperatingSystem.LINUX: {KeyCode.Meta: "Super"},
        OperatingSystem.MACOS: {
            KeyCode.Ctrl: "Control",
            KeyCode.Alt: "Option",
            KeyCode.Meta: "Cmd",
        },
    }
```